### PR TITLE
Fixes in lwjgl3

### PR DESCRIPTION
**Pull request by JavaSaBr**
When I switched my jME3-SpaceShift-Editor to lwjgl3, I got several problems.
1. The input of chars doesn't work for UI.
2. Mouse wheel doesn't works.
3. NPE on start of application.

### CODE REVIEW

File: jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public void invoke(long window, double xpos, double ypos) {
         glfwSetScrollCallback(context.getWindowHandle(), scrollCallback = new GLFWScrollCallback() {
             @Override
             public void invoke(final long window, final double xOffset, final double yOffset) {
-                onWheelScroll(window, xOffset, yOffset);
+                onWheelScroll(window, xOffset, yOffset * 120);
```
Comment (DannyJo) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: Why 120? Feels a bit magic, could we put it as a constant with a proper name?

Comment (JavaSaBr) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: I compared this value with LWJGL2.

Comment (kvainer) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: I see it being used for Linux and Mac, but I cannot find it used on Windows:
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/LinuxMouse.java#L48
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/MacOSXNativeMouse.java#L53

Comment (shadowislord) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: I see it being used for Linux and Mac, but I cannot find it used on Windows:
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/LinuxMouse.java#L48
https://github.com/LWJGL/lwjgl/blob/master/src/java/org/lwjgl/opengl/MacOSXNativeMouse.java#L53

Comment (JavaSaBr) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: I tested my Editor on Ubuntu 14.04 and Windows 7. This fix works fine.

Comment (shadowislord) [inline jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java]: Can you move the number into a constant field? Then I'll merge this.

**Comment by JavaSaBr** [comment]
Updated

**Pull request merged**